apps/judicial-system/web/src/components/Inputs/InputNationalId.tsx

```
@@ -88,6 +88,7 @@ const InputNationalId: FC<Props> = (props) => {
       replacement={{ _: /\d/ }}
       component={Input}
       value={inputValue ?? value}
+      data-testid="inputNationalId"
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)